chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -835,6 +848,8 @@ pub enum ProcessCommitError {
     CommitError,
     #[error("decrypted path secret don't match the public key")]
     PathSecretPublicKeyDontMatch,
+    #[error("cached secret key don't match commit leaf keypackage")]
+    LeafSecretPublicKeyDontMatch,
```

Comment:
seems this error is not used?

Reply:
yes, let me remove it, this error case is now keypackage verify error.

Reply:
done